src/main/java/org/spongepowered/common/entity/living/human/EntityHuman.java

```diff
@@ -507,7 +507,7 @@ public void attackEntityWithRangedAttack(EntityLivingBase target, float p_82196_
 
         final ItemStack itemstack = this.getHeldItem(EnumHand.OFF_HAND);
 
-        if (itemstack != null && itemstack.getItem() == Items.TIPPED_ARROW) {
+        if (!itemstack.isEmpty() && itemstack.getItem() == Items.TIPPED_ARROW) {
```

Comment:
An ItemStack cannot be empty unless it's Item is ItemAir. You can remove the isEmpty altogether.